great_expectations/data_context/store/html_site_store.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, store_backend=None, runtime_environment=None):
                 "module_name": module_name,
                 "filepath_prefix": filepath_prefix,
                 "filepath_suffix": filepath_suffix,
+                "dont_add_store_backend_id": True,
```

Comment:
another nit --> I have a pet peeve against using a contraction in code since the apostrophe is omitted. Can you rename this parameter to something like `suppress_store_backend_id`?

Reply:
Done! Thanks for the suggestion, `suppress_store_backend_id` sounds much better.